osschallenge/static/osschallenge/stylesheet.css

```diff
@@ -736,3 +736,46 @@ table.table.table--striped.table--hover.table--bordered tbody tr td a{
     text-align: left;
     width: 200px;
 }
+
+div.fb-share-button {
```

Comment:
Could this class and the tweet-button class be merged? Only the `background-color` is different. Let us talk about this one.